### DISCUSSION TITLE
Open Radar AGU Abstract

### DISCUSSION THREAD

**Discussion by mgrover1** [openradar/erad2022#40] [Ideas]
Hey ya'll (@openradar/erad2022-team)- I am putting together an abstract for the Open Science session at AGU, and would like to present on the Open Radar community, with one of the highlights being this workshop. I would like to add ya'll as co-authors on the abstract, and solicit feedback from this group!

[Link to the abstract draft](https://docs.google.com/document/d/1QNtDF8gjQXPHWp6RjoFxReu2E62tbLWvMv-nNuK1rJA/edit?usp=sharing)

I plan on submitting early next week, so I would like to look at comments by **Monday August 1** at the latest.

**Comment by kmuehlbauer** [comment]
Thanks Max, you can count me in. I could elaborate on anything wradlib is involved but also on jupyter ecosystem and docker stuff. Unfortunately I'll not be available before August, 8th.

**Reply by mgrover1** [reply]
Sounds good!

**Comment by DanielMichelson** [comment]
Hi Max et al,

Please sign me up. I'd appreciate a mention of BALTRAD somewhere, perhaps
at the beginning of the second paragraph.
If I were to suggest any change, it would be to the 4th sentence in the
first paragraph that I would simplify and redirect:
"This is in contrast to a closed-ecosystem, where proprietary software and
data prevent inclusive, transparent and reproducible Open Science."

Thanks, Daniel

**Comment by mgrover1** [comment]
Thank you for the feedback - I apologize for not including it in there
originally. I’ll add it in the second paragraph!

Thanks

Max

**Reply by jfigui** [reply]
Hi @mgrover1 ,
Full support to this initiative. If you think it is pertinent you can present Pyrad as part of the Py-ART ecosystem (as PyDDA is). We can create Pyrad config files to automatize one of the Py-ART notebooks that you present during the conference.
Pyrad PIs are Daniel Wolfensberger and myself. I let to you to decide whether we should be co-authors.

Jordi

**Reply by mgrover1** [reply]
@jfigui that sounds like a great plan! I will add you all as well.
